converter/src/test/java/gov/cms/qpp/conversion/encode/MeasureDataEncoderTest.java

```diff
@@ -18,31 +18,31 @@
 
 	@Test
 	void testDenominator() throws EncodeException {
-		Node measureDataNode = setUpMeasureDataNode(SubPopulations.DENOM);
+		Node measureDataNode = setUpMeasureDataNode(SubPopulationLabel.DENOM.name());
```

Comment:
Can these tests be rolled up to a parameterized test?

Reply:
yes